logictest: bump SET CLUSTER TIMEOUT wait setting for tests

otan opened the pull request:
This will alleviate the `setting updated but timed out waiting to read
new value` we see somewhat frequently in logic tests for multi-node
clusters.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/69227)
<!-- Reviewable:end -->

arulajmani reviewed (approved):
Was 10s an unreasonable timeout for our tests earlier or is there more to investigate here?

> Reviewed 3 of 3 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/69227#-:-MhnG49VDR1ZHeT6s81J:b-rkx5s8)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @otan)



<!-- Sent from Reviewable.io -->

otan commented:
there's probably more, but in the meantime...

bors r=arulajmani

andreimatei reviewed (commented):
Can these test failures be reproduced? It sure sounds to me like settings should propagate within 10s. Perhaps we have a bigger gossip tuning task to do.
@otan , do I have your word that you're doing the "more"? Do you mind opening an issue if there isn't one and putting a TODO(issue) in the code? Sorry to make a stink, but I really don't want this patch to stand for long.

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/69227#-:-MhpDs_24BWad5NXewFl:b-kvdzis)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @otan)



<!-- Sent from Reviewable.io -->

otan commented:
bors r-

yep, let me fix that up.

otan commented:
@andreimatei example of "more": https://github.com/cockroachdb/cockroach/issues/69233

otan commented:
i filed #69272 in the meantime

otan committed:
logictest: bump SET CLUSTER TIMEOUT wait setting for tests

This will alleviate the `setting updated but timed out waiting to read
new value` we see somewhat frequently in logic tests for multi-node
clusters.

Release note: None

otan commented:
i'd rather this merge than #69322. 

bors r=arulajmani
